platforms/mixin-lifecycle/README.md

```diff
@@ -1,3 +1,3 @@
 # mixin-lifecycle
 
-This project contains lifecycle mixins shared between Fabric and Quilt.
\ No newline at end of file
+This project contains lifecycle mixins shared between Fabric and formerly Quilt.
```

Comment:
Similarly to above change this line to "This project contains lifecycle mixins for Fabric."  Seems redundant to mention Quilt if it's being removed.